analytics/local.env

```diff
@@ -33,7 +33,7 @@ MB_DB_PASS=secret123
 MB_DB_HOST=grants-analytics-db
 
 # Metabase API configuration
-MB_API_URL=http://metabase-dev-710651776.us-east-1.elb.amazonaws.com/api
+MB_API_URL=https://data.simpler.grants.gov/api
```

Comment:
Nice! Excited that we have a dedicated URL now.

This is the prod instance though right?